### PR TITLE
Add Replicate demo and API

### PR DESCRIPTION
**Pull request by chenxwh**
Hey @Jingkang50! 👋 

This pull request adds another demo to your model, which runs inside a Docker environment using an open source tool called [Cog](https://github.com/replicate/cog) to make this process easier. View it here: https://replicate.com/cjwbw/openpsg

Replicate also have an API, so people can easily run your model from their code:

```
import replicate
model = replicate.models.get("cjwbw/openpsg")
model.predict(image="...")
```

If you'd like to modify the Replicate page (e.g. [Example](https://replicate.com/cjwbw/openpsg/examples) Gallery), let me know and I can transfer ownership to your account.

In case you're wondering who I am, I'm from [Replicate](https://replicate.com/home), where we're trying to make machine learning reproducible. We got frustrated that we couldn't run all the really interesting ML work being done. So, we're going round implementing models we like. 😊

### CODE REVIEW

**Comment by c-liangyu** [comment]
LGTM

**Review by c-liangyu — COMMENTED** [review]
Do we need to display the first ouput.png? I hope this segmentation graph is not too messy to confuse users.

**Comment by Jingkang50** [comment]
Thank you sooo much @chenxwh for helping to build the cool demo! Your work and your organization Replicate sounds super cool! Let us review your PR then!

**Comment by Jingkang50** [comment]
@c-liangyu Ideally the first output.png shall be replaced with a GIF that you are currently developing. How about finish it first and kindly ask @chenxwh to revise a little bit? Thanks!

**Comment by chenxwh** [comment]
Hi @Jingkang50, 

Thanks for the interest in our work! This PR is more like a demonstration how models can be hosted on Replicate, authors are more than welcome to take the ownership of the page and modify the demo however they like :) I can transfer the page to your account on Replicate If you like, then with cog installed you can update any further changes to the demo anytime. Let me know!

File: README.md [hunk 1]
```diff
@@ -28,6 +28,11 @@
   &nbsp;&nbsp;&nbsp;
   <a href="https://join.slack.com/t/psgdataset/shared_invite/zt-1d14sdkw3-59pJdrp6gLAHuBObPL91qw" target='_blank'>
     <img src="https://img.shields.io/badge/Forum-Slack-4c1448?style=flat-square">
+    &nbsp;&nbsp;&nbsp;
+  <a href="https://replicate.com/cjwbw/openpsg" target='_blank'>
+    <img src="https://img.shields.io/badge/Replicate-Demo & Cloud API-1b82c2?style=flat-square">
+
+    [![Replicate](https://replicate.com/kimgeonung/bigcolor/badge)](https://replicate.com/kimgeonung/bigcolor)
```
Comment (c-liangyu) [inline README.md:35]: Hi! What is this line for?

Comment (chenxwh) [reply README.md]: Ah sorry was accidental (that was our usual badge so I was copy pasting for the new url but then realised you have a different template but forgot to remove the old one) - have removed now!

**Review by Jingkang50 — APPROVED** [review]

**Comment by Jingkang50** [comment]
Thank you @chenxwh for your contribution! Really appreciate it!

**Pull request merged**